student-store-ui/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import "./Footer.css"
+
+export default function Footer() {
+  return (
+      <footer class="footer">
+        <div className="content">
+            <div className="top">
+                <div className="links">
+                    <div className="link-column">
+                        <h4>Categories</h4>
+                        <ul>
+                            <li>All Categories</li>
+                            <li>Clothing</li>
+                            <li>Food</li>
+                            <li>Accessories</li>
+                            <li>Tech</li>
+                        </ul>
+                    </div>
+                    <div className="link-column">
+                        <h4>Companies</h4>
+                        <ul>
+                            <li>All Categories</li>
+                            <li>Clothing</li>
+                            <li>Food</li>
+                            <li>Accessories</li>
+                            <li>Tech</li>
+                        </ul>
+                    </div>
+                    <div className="link-column">
+                        <h4>Support</h4>
+                        <ul>
+                            <li>All Categories</li>
+                            <li>Clothing</li>
+                            <li>Food</li>
+                            <li>Accessories</li>
+                            <li>Tech</li>
+                        </ul>
+                    </div>
+                    <div className="link-column">
+                        <h4>Account</h4>
+                        <ul>
+                            <li>All Categories</li>
+                            <li>Clothing</li>
+                            <li>Food</li>
+                            <li>Accessories</li>
+                            <li>Tech</li>
+                        </ul>
+                    </div>
+                    <div className="link-column">
+                        <h4>Socials</h4>
+                        <ul>
+                            <li>All Categories</li>
+                            <li>Clothing</li>
+                            <li>Food</li>
+                            <li>Accessories</li>
+                            <li>Tech</li>
+                        </ul>
```

Comment:
can abstract to functional component

i.e FooterColumn

pass it in a header name as prop

then in the render fxn loop through array of header vals to render

i.e 
```
{['Categories', 'Companies', ...].map( (val) => <FooterColumn header={val} />  }
```

to prevent having duplicated list code :d